code_files/main.py

```diff
@@ -0,0 +1,33 @@
+import argparse
```

Comment:
Let's move this main file out of this folder (code_files) .

Reply:
okay